docs(gpu):scratch-storage (#2146)
scaleway/docs-content: compute/gpu/concepts.mdx

## Scratch storage

Scratch storage refers to a type of temporary storage space used in computing systems, particularly in high-performance computing (HPC) environments and supercomputers. 
It is used for short-term cache storage of data that is being actively processed, but it is not meant for long-term or permanent storage.
Scratch storage is an ephemeral storage solution that does not support features like snapshots, backups, or restores.

<Message type="note">

>   Scratch storage does not survive stopping the server, doing a full stop/start cycle will erase the scratch data. However, doing a simple reboot or using the stop in place function will keep the data.

Reviewer: ```suggestion
  Scratch storage does not prevent your data from being lost when stopping the server. Doing a full stop/start cycle erases scratch data. However, rebooting or using the stop in place function will keep the data.
```

>   Scratch storage does not survive stopping the server, doing a full stop/start cycle will erase the scratch data. However, doing a simple reboot or using the stop in place function will keep the data.

Reviewer 2: ```suggestion
  Scratch storage does not survive once the server is stopped: doing a full stop/start cycle will erase the scratch data. However, doing a simple reboot or using the stop in place function will keep the data.
```
